test/narrativebuilder.cc

```diff
@@ -8879,488 +8828,7 @@ void TestFormExitLeftInstruction() {
-
-  // BuildTransit_1_stop_count_4_miles_cs_CZ
-  suite.test(TEST_CASE(TestBuildTransit_1_stop_count_4_miles_cs_CZ));
-
-  // BuildTransit_1_stop_count_8_miles_cs_CZ
-  suite.test(TEST_CASE(TestBuildTransit_1_stop_count_8_miles_cs_CZ));
-
-  // BuildTransitTransfer_0_no_name_miles_en_US
-  suite.test(TEST_CASE(TestBuildTransitTransfer_0_no_name_miles_en_US));
-
-  // BuildTransitTransfer_0_miles_en_US
-  suite.test(TEST_CASE(TestBuildTransitTransfer_0_miles_en_US));
-
-  // BuildTransitTransfer_1_miles_en_US
-  suite.test(TEST_CASE(TestBuildTransitTransfer_1_miles_en_US));
-
-  // BuildTransitRemainOn_0_no_name_miles_en_US
-  suite.test(TEST_CASE(TestBuildTransitRemainOn_0_no_name_miles_en_US));
-
-  // BuildTransitRemainOn_0_miles_en_US
-  suite.test(TEST_CASE(TestBuildTransitRemainOn_0_miles_en_US));
-
-  // BuildTransitRemainOn_1_miles_en_US
-  suite.test(TEST_CASE(TestBuildTransitRemainOn_1_miles_en_US));
-
-  // BuildPostTransitConnectionDestination_0_miles_en_US
-  suite.test(TEST_CASE(TestBuildPostTransitConnectionDestination_0_miles_en_US));
-
-  // BuildPostTransitConnectionDestination_1_miles_en_US
-  suite.test(TEST_CASE(TestBuildPostTransitConnectionDestination_1_miles_en_US));
-
-  // BuildPostTransitConnectionDestination_2_miles_en_US
-  suite.test(TEST_CASE(TestBuildPostTransitConnectionDestination_2_miles_en_US));
-
-  // BuildPostTransitConnectionDestination_4_miles_en_US
-  suite.test(TEST_CASE(TestBuildPostTransitConnectionDestination_4_miles_en_US));
-
-  // BuildPostTransitConnectionDestination_5_miles_en_US
-  suite.test(TEST_CASE(TestBuildPostTransitConnectionDestination_5_miles_en_US));
-
-  // BuildPostTransitConnectionDestination_6_miles_en_US
-  suite.test(TEST_CASE(TestBuildPostTransitConnectionDestination_6_miles_en_US));
-
-  // BuildPostTransitConnectionDestination_8_miles_en_US
-  suite.test(TEST_CASE(TestBuildPostTransitConnectionDestination_8_miles_en_US));
-
-  // BuildPostTransitConnectionDestination_9_miles_en_US
-  suite.test(TEST_CASE(TestBuildPostTransitConnectionDestination_9_miles_en_US));
-
-  // BuildPostTransitConnectionDestination_10_miles_en_US
-  suite.test(TEST_CASE(TestBuildPostTransitConnectionDestination_10_miles_en_US));
-
-  // BuildPostTransitConnectionDestination_16_miles_en_US
-  suite.test(TEST_CASE(TestBuildPostTransitConnectionDestination_16_miles_en_US));
-
-  // BuildPostTransitConnectionDestination_17_miles_en_US
-  suite.test(TEST_CASE(TestBuildPostTransitConnectionDestination_17_miles_en_US));
-
-  // BuildPostTransitConnectionDestination_18_miles_en_US
-  suite.test(TEST_CASE(TestBuildPostTransitConnectionDestination_18_miles_en_US));
-
-  // BuildVerbalMultiCue_0_miles_en_US
-  suite.test(TEST_CASE(TestBuildVerbalMultiCue_0_miles_en_US));
-
-  // End of the build phrase tests
-  //////////
-
-  return suite.tear_down();
+int main(int argc, char* argv[]) {
+  testing::InitGoogleTest(&argc, argv);
+  return RUN_ALL_TESTS();
```

Comment:
i hope and pray that you did not edit this insane file by hand

Reply:
of course not :) regexps helped here. But this is not the most insane file either